src/test/java/core/basesyntax/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String SOURCE_FILE = "src/main/java/resources/filetest";
+    private final FileReaderImpl fileService = new FileReaderImpl();
+    private final List<String> expected = new ArrayList<>();
+
+    @Before
+    public void setUp() throws Exception {
+        expected.add("b,banana,20");
+        expected.add("b,apple,100");
+        expected.add("s,banana,100");
+        expected.add("p,banana,13");
+        expected.add("r,apple,10");
+        expected.add("p,apple,20");
+        expected.add("p,banana,5");
+        expected.add("s,banana,50");
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void readFile_Null_notOk() {
+        fileService.read(null);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void readFile_incorrectPath_NotOk() {
+        fileService.read("src/main/java/resourcesee/filetest");
+    }
+
+    @Test
+    public void readFile_validFileName_Ok() {
+        assertEquals(expected, fileService.read(SOURCE_FILE));
```

Comment:
If you use expected variable only in one test, i don't think we should make it as class variable, lets make it local